src/docs/gold_standard_slot_examples.md

```diff
@@ -0,0 +1,444 @@
+  in_subset:
+    - nucleic acid sequence source
+  keywords:
+    - fraction
+    - size
+  string_serialization: "{float}-{float} {unit}"
+  slot_uri: MIXS:0000017
+```
+
+This example shows a slot that records a numeric measurement value with a specific unit of measurement.
+The value is a float, however the particular unit can be defined by the user (or interpreted by the implementation of the schema) as a pre-defined string format, as defined in the `settings:` section of the schema (like with the `structured_pattern:` attribute).
+In this example the measurement is made as a range of values, however the value itself could also be a single value such as `{float} {unit}`.
+In this particular case, the `string_serialization:` attribute is used to define the format of the string in a different way to the `structured_pattern:` attribute.
+
+<!--
+JFY comment: someone else will need to explain why the string_serialization is used here rather than a structured_pattern
+-->
+
+<!--
+JFY comment: I don't like this so much as:
+
+- single example
+- is a value range not a single value
+- missing range? (but possibly assumed default)
+-->
+
+### Structured text (ontology term)
+
+An example of a close-to gold standard _Structured text (ontology term)_ slot is [`env_medium`](https://genomicsstandardsconsortium.github.io/mixs/0000014/)
+
+```yaml
+env_medium:
+  description: "Report the environmental material(s) immediately surrounding the
+    sample or specimen at the time of sampling. We recommend using subclasses of
+    'environmental material' (http://purl.obolibrary.org/obo/ENVO_00010483). EnvO
+    documentation about how to use the field: https://github.com/EnvironmentOntology/envo/wiki/Using-ENVO-with-MIxS
+    . Terms from other OBO ontologies are permissible as long as they reference
+    mass/volume nouns (e.g. air, water, blood) and not discrete, countable entities
+    (e.g. a tree, a leaf, a table top)"
+  title: environmental medium
+  examples:
+    - value: bluegrass field soil [ENVO:00005789]
+  in_subset:
+    - environment
+  keywords:
+    - environmental
+  slot_uri: MIXS:0000014
+  range: string
+  required: true
+  pattern: ^([^\s-]{1,2}|[^\s-]+.+[^\s-]+) \[[a-zA-Z]{2,}:[a-zA-Z0-9]\d+\]$
+  structured_pattern:
+    syntax: ^{termLabel} \[{termID}\]$
+    interpolated: true
+    partial_match: true
+```
+
+This example shows a specific type of structured text slot that is commonly used across in MIxS, where the value refers to a specific ontology term.
+This particular pattern has the pattern of an ontology term label (a string) plus a unique identifier (a string) in square brackets.
+Implementers of the schema will often use such slots to perform API lookups to the ontology term against the ontology database, thus the pattern structure is critical when defining an ontology term-based metadata slot
+
+<!--
+JFY comment: I don't like this so much as:
+
+- Why pattern AND structured_pattern?
+-->
+
+### URL
```

Comment:
The structured pattern for this, and I think most slots that allow for URLs is "Structured pattern: ^{PMID}|{DOI}|{URL}$"

So an ID/DOI is also accepted... I wonder if we can refer to this as something else.. source? reference? IDK! But let's expand beyond URL and pick a slot with a URL, DOI, or PMID example.

Reply:
I don't follow @mslarae13 , sorry!

Do you mean a slot example with multiple examples? Or you mean update the header/title?